### DISCUSSION TITLE
did you include Rostino roasters inside the program ?

### DISCUSSION THREAD

**Discussion by cevo1988** [artisan-roaster-scope/artisan#1287] [General]
Hello sirs
I would like to ask if your program is compatible with [rostinoroasters.com](url) ?
www.rostinoroasters.com

**Comment by poundy** [comment]
Here's the published list of Artisan supported machines. https://artisan-scope.org/machines/ 

Please note, this list is the machines that have been funded to be incorporated in simple setup in Artisan - this is not the comprehensive list of machines that potentially could work with Artisan.  There are many roasting machines not on that list that people have individually set up, bith commercially produced roasters or more home-brew style.  Mostly, vendors who want to have a roast monitoring/management integration are engaging the Artisan team to properly develop and integrate their equipment into the Artisan ecosystem to make their machines more palatable to potential purchasers. If you have one of these roasters you should approach them to   get Artisan to support their roaster.
